react/features/shared-iframe/components/SharedIFrameButtonContainer.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import React from 'react';
+
+import { translate } from '../../base/i18n';
+import { connect } from '../../base/redux';
+
+import SharedIFrameButton from './SharedIFrameButton';
+
+type Props = AbstractButtonProps & {
```

Comment:
Please make sure to import these props.

Reply:
01aaf7adc964d7bfd75f23bb02e2f1e80937c56a